### PR TITLE
[F] Improvements to layout footer

### PR DESCRIPTION
**Pull request by SMaxOwok**
- Social links are configurable by installation
- Press logo is displayed in footer and links to press url
- Manifold label is displayed as subfooter
- Implemented a contact form.  Recipient address is configurable by installation.

### CODE REVIEW

**Comment by SMaxOwok** [comment]
@blnkt, @zdavis I moved the press site URL setting into the theme tab, so that it is with the logo that it'll link to.  I left if as a general setting in the model though.  Let me know if you feel it should be moved to the theme settings on the model or moved back into the general settings form.

**Comment by zdavis** [comment]
@SMaxOwok please resolve conflicts.

**Comment by zdavis** [comment]
I'm not sure the privacy link is working as expected.

image

**Comment by zdavis** [comment]
Textarea in contact form should be taller. About like this:

image

**Comment by zdavis** [comment]
Email appears to send even if I enter an email that's not even remotely valid.

**Comment by SMaxOwok** [comment]
@zdavis re: privacy policy, that was implemented as part of https://github.com/ManifoldScholar/manifold/issues/868.  I suppose that field should give some explanation of where it will appear.

I'll add some validation to the email.

**Comment by SMaxOwok** [comment]
Updated

File: api/app/mailers/contact_mailer.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class ContactMailer < ApplicationMailer
+  def contact_message(full_name, email, message)
+    installation_contact = @settings.general[:contact_email]
+    return unless installation_contact.present?
```
Comment (zdavis) [inline api/app/mailers/contact_mailer.rb]: We should also return if email.blank?, no?

Comment (zdavis) [reply api/app/mailers/contact_mailer.rb]: Derps.

**Pull request merged**